src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/InvitedPersonForCreateCommand.cs

```diff
@@ -5,17 +5,14 @@ namespace Equinor.ProCoSys.IPO.Command.InvitationCommands.CreateInvitation
     public class InvitedPersonForCreateCommand : IInvitedPersonForCommand
     {
         public InvitedPersonForCreateCommand(
-            Guid? azureOid,
-            string email,
+            Guid azureOid,
             bool required)
         {
             AzureOid = azureOid;
-            Email = email;
             Required = required;
         }
      
-        public Guid? AzureOid { get; }
-        public string Email { get; }
+        public Guid AzureOid { get; }
```

Comment:
line 122 in CreateInvitationCommandHandler still has a check if AzureOid != null